From 810b0b4bb8aabb3318b1d92064183b0c2a23937a Mon Sep 17 00:00:00 2001
From: Valentine Barshak <valentine.barshak@cogentembedded.com>
Date: Mon, 19 Nov 2018 00:46:09 +0300
Subject: [PATCH 02/21] configs: rcar-gen3: Enable askenv command

This enables "askenv" command for R-Car Gen3 boards
which can be used in U-Boot scripts for user input.

Signed-off-by: Valentine Barshak <valentine.barshak@cogentembedded.com>
---
 configs/r8a7795_salvator-x_defconfig       | 1 +
 configs/r8a7795_salvator-xs-2x2g_defconfig | 1 +
 configs/r8a7795_salvator-xs-4x2g_defconfig | 1 +
 configs/r8a7795_salvator-xs_defconfig      | 1 +
 configs/r8a7795_ulcb-4x2g_defconfig        | 1 +
 configs/r8a7795_ulcb_defconfig             | 1 +
 configs/r8a77965_salvator-x_defconfig      | 1 +
 configs/r8a77965_salvator-xs_defconfig     | 1 +
 configs/r8a77965_ulcb_defconfig            | 1 +
 configs/r8a7796_salvator-x_defconfig       | 1 +
 configs/r8a7796_salvator-xs-2x4g_defconfig | 1 +
 configs/r8a7796_salvator-xs_defconfig      | 1 +
 configs/r8a7796_ulcb-2x4g_defconfig        | 1 +
 configs/r8a7796_ulcb_defconfig             | 1 +
 configs/r8a77970_eagle_defconfig           | 1 +
 configs/r8a77980_condor_defconfig          | 1 +
 configs/r8a77990_ebisu-4d_defconfig        | 1 +
 configs/r8a77990_ebisu_defconfig           | 1 +
 configs/r8a77995_draak_defconfig           | 1 +
 19 files changed, 19 insertions(+)

diff --git a/configs/r8a7795_salvator-x_defconfig b/configs/r8a7795_salvator-x_defconfig
index e6d8dfe..2ced2cf 100644
--- a/configs/r8a7795_salvator-x_defconfig
+++ b/configs/r8a7795_salvator-x_defconfig
@@ -16,6 +16,7 @@ CONFIG_DEFAULT_FDT_FILE="r8a7795-salvator-x.dtb"
 CONFIG_VERSION_VARIABLE=y
 CONFIG_HUSH_PARSER=y
 CONFIG_CMD_BOOTZ=y
+CONFIG_CMD_ASKENV=y
 CONFIG_CMD_GPIO=y
 CONFIG_CMD_I2C=y
 CONFIG_CMD_MMC=y
diff --git a/configs/r8a7795_salvator-xs-2x2g_defconfig b/configs/r8a7795_salvator-xs-2x2g_defconfig
index 8cb4896..315d446 100644
--- a/configs/r8a7795_salvator-xs-2x2g_defconfig
+++ b/configs/r8a7795_salvator-xs-2x2g_defconfig
@@ -16,6 +16,7 @@ CONFIG_DEFAULT_FDT_FILE="r8a7795-salvator-xs-2x2g.dtb"
 CONFIG_VERSION_VARIABLE=y
 CONFIG_HUSH_PARSER=y
 CONFIG_CMD_BOOTZ=y
+CONFIG_CMD_ASKENV=y
 CONFIG_CMD_GPIO=y
 CONFIG_CMD_I2C=y
 CONFIG_CMD_MMC=y
diff --git a/configs/r8a7795_salvator-xs-4x2g_defconfig b/configs/r8a7795_salvator-xs-4x2g_defconfig
index f916800..82177ac 100644
--- a/configs/r8a7795_salvator-xs-4x2g_defconfig
+++ b/configs/r8a7795_salvator-xs-4x2g_defconfig
@@ -16,6 +16,7 @@ CONFIG_DEFAULT_FDT_FILE="r8a7795-salvator-xs-4x2g.dtb"
 CONFIG_VERSION_VARIABLE=y
 CONFIG_HUSH_PARSER=y
 CONFIG_CMD_BOOTZ=y
+CONFIG_CMD_ASKENV=y
 CONFIG_CMD_GPIO=y
 CONFIG_CMD_I2C=y
 CONFIG_CMD_MMC=y
diff --git a/configs/r8a7795_salvator-xs_defconfig b/configs/r8a7795_salvator-xs_defconfig
index ed3b4b1..b05deab 100644
--- a/configs/r8a7795_salvator-xs_defconfig
+++ b/configs/r8a7795_salvator-xs_defconfig
@@ -16,6 +16,7 @@ CONFIG_DEFAULT_FDT_FILE="r8a7795-salvator-xs.dtb"
 CONFIG_VERSION_VARIABLE=y
 CONFIG_HUSH_PARSER=y
 CONFIG_CMD_BOOTZ=y
+CONFIG_CMD_ASKENV=y
 CONFIG_CMD_GPIO=y
 CONFIG_CMD_I2C=y
 CONFIG_CMD_MMC=y
diff --git a/configs/r8a7795_ulcb-4x2g_defconfig b/configs/r8a7795_ulcb-4x2g_defconfig
index 65fa387..8e21367 100644
--- a/configs/r8a7795_ulcb-4x2g_defconfig
+++ b/configs/r8a7795_ulcb-4x2g_defconfig
@@ -17,6 +17,7 @@ CONFIG_DEFAULT_FDT_FILE="r8a7795-h3ulcb-4x2g.dtb"
 CONFIG_VERSION_VARIABLE=y
 CONFIG_HUSH_PARSER=y
 CONFIG_CMD_BOOTZ=y
+CONFIG_CMD_ASKENV=y
 CONFIG_CMD_GPIO=y
 CONFIG_CMD_I2C=y
 CONFIG_CMD_MMC=y
diff --git a/configs/r8a7795_ulcb_defconfig b/configs/r8a7795_ulcb_defconfig
index 7acd725..5d388f8 100644
--- a/configs/r8a7795_ulcb_defconfig
+++ b/configs/r8a7795_ulcb_defconfig
@@ -17,6 +17,7 @@ CONFIG_DEFAULT_FDT_FILE="r8a7795-h3ulcb.dtb"
 CONFIG_VERSION_VARIABLE=y
 CONFIG_HUSH_PARSER=y
 CONFIG_CMD_BOOTZ=y
+CONFIG_CMD_ASKENV=y
 CONFIG_CMD_GPIO=y
 CONFIG_CMD_I2C=y
 CONFIG_CMD_MMC=y
diff --git a/configs/r8a77965_salvator-x_defconfig b/configs/r8a77965_salvator-x_defconfig
index 8d42ad2..24804e2 100644
--- a/configs/r8a77965_salvator-x_defconfig
+++ b/configs/r8a77965_salvator-x_defconfig
@@ -16,6 +16,7 @@ CONFIG_DEFAULT_FDT_FILE="r8a77965-salvator-x.dtb"
 CONFIG_VERSION_VARIABLE=y
 CONFIG_HUSH_PARSER=y
 CONFIG_CMD_BOOTZ=y
+CONFIG_CMD_ASKENV=y
 CONFIG_CMD_GPIO=y
 CONFIG_CMD_I2C=y
 CONFIG_CMD_MMC=y
diff --git a/configs/r8a77965_salvator-xs_defconfig b/configs/r8a77965_salvator-xs_defconfig
index 3ff7a6a..d94003a 100644
--- a/configs/r8a77965_salvator-xs_defconfig
+++ b/configs/r8a77965_salvator-xs_defconfig
@@ -16,6 +16,7 @@ CONFIG_DEFAULT_FDT_FILE="r8a77965-salvator-xs.dtb"
 CONFIG_VERSION_VARIABLE=y
 CONFIG_HUSH_PARSER=y
 CONFIG_CMD_BOOTZ=y
+CONFIG_CMD_ASKENV=y
 CONFIG_CMD_GPIO=y
 CONFIG_CMD_I2C=y
 CONFIG_CMD_MMC=y
diff --git a/configs/r8a77965_ulcb_defconfig b/configs/r8a77965_ulcb_defconfig
index fcd367a..f72e8fb 100644
--- a/configs/r8a77965_ulcb_defconfig
+++ b/configs/r8a77965_ulcb_defconfig
@@ -17,6 +17,7 @@ CONFIG_DEFAULT_FDT_FILE="r8a77965-m3nulcb.dtb"
 CONFIG_VERSION_VARIABLE=y
 CONFIG_HUSH_PARSER=y
 CONFIG_CMD_BOOTZ=y
+CONFIG_CMD_ASKENV=y
 CONFIG_CMD_GPIO=y
 CONFIG_CMD_I2C=y
 CONFIG_CMD_MMC=y
diff --git a/configs/r8a7796_salvator-x_defconfig b/configs/r8a7796_salvator-x_defconfig
index 5d7fcc5..0a2299f 100644
--- a/configs/r8a7796_salvator-x_defconfig
+++ b/configs/r8a7796_salvator-x_defconfig
@@ -16,6 +16,7 @@ CONFIG_DEFAULT_FDT_FILE="r8a7796-salvator-x.dtb"
 CONFIG_VERSION_VARIABLE=y
 CONFIG_HUSH_PARSER=y
 CONFIG_CMD_BOOTZ=y
+CONFIG_CMD_ASKENV=y
 CONFIG_CMD_GPIO=y
 CONFIG_CMD_I2C=y
 CONFIG_CMD_MMC=y
diff --git a/configs/r8a7796_salvator-xs-2x4g_defconfig b/configs/r8a7796_salvator-xs-2x4g_defconfig
index 07a7bb7..2d5e7a0 100644
--- a/configs/r8a7796_salvator-xs-2x4g_defconfig
+++ b/configs/r8a7796_salvator-xs-2x4g_defconfig
@@ -19,6 +19,7 @@ CONFIG_MULTI_DTB_FIT_USER_DEFINED_AREA=y
 CONFIG_VERSION_VARIABLE=y
 CONFIG_HUSH_PARSER=y
 CONFIG_CMD_BOOTZ=y
+CONFIG_CMD_ASKENV=y
 CONFIG_CMD_GPIO=y
 CONFIG_CMD_I2C=y
 CONFIG_CMD_MMC=y
diff --git a/configs/r8a7796_salvator-xs_defconfig b/configs/r8a7796_salvator-xs_defconfig
index 2c906c9..d38dab4 100644
--- a/configs/r8a7796_salvator-xs_defconfig
+++ b/configs/r8a7796_salvator-xs_defconfig
@@ -19,6 +19,7 @@ CONFIG_MULTI_DTB_FIT_USER_DEFINED_AREA=y
 CONFIG_VERSION_VARIABLE=y
 CONFIG_HUSH_PARSER=y
 CONFIG_CMD_BOOTZ=y
+CONFIG_CMD_ASKENV=y
 CONFIG_CMD_GPIO=y
 CONFIG_CMD_I2C=y
 CONFIG_CMD_MMC=y
diff --git a/configs/r8a7796_ulcb-2x4g_defconfig b/configs/r8a7796_ulcb-2x4g_defconfig
index 55a660e..36a164c 100644
--- a/configs/r8a7796_ulcb-2x4g_defconfig
+++ b/configs/r8a7796_ulcb-2x4g_defconfig
@@ -18,6 +18,7 @@ CONFIG_DEFAULT_FDT_FILE="r8a7796-m3ulcb-2x4g.dtb"
 CONFIG_VERSION_VARIABLE=y
 CONFIG_HUSH_PARSER=y
 CONFIG_CMD_BOOTZ=y
+CONFIG_CMD_ASKENV=y
 CONFIG_CMD_GPIO=y
 CONFIG_CMD_I2C=y
 CONFIG_CMD_MMC=y
diff --git a/configs/r8a7796_ulcb_defconfig b/configs/r8a7796_ulcb_defconfig
index c765c93..3447d6d 100644
--- a/configs/r8a7796_ulcb_defconfig
+++ b/configs/r8a7796_ulcb_defconfig
@@ -17,6 +17,7 @@ CONFIG_DEFAULT_FDT_FILE="r8a7796-m3ulcb.dtb"
 CONFIG_VERSION_VARIABLE=y
 CONFIG_HUSH_PARSER=y
 CONFIG_CMD_BOOTZ=y
+CONFIG_CMD_ASKENV=y
 CONFIG_CMD_GPIO=y
 CONFIG_CMD_I2C=y
 CONFIG_CMD_MMC=y
diff --git a/configs/r8a77970_eagle_defconfig b/configs/r8a77970_eagle_defconfig
index a303796..0aac4ad 100644
--- a/configs/r8a77970_eagle_defconfig
+++ b/configs/r8a77970_eagle_defconfig
@@ -18,6 +18,7 @@ CONFIG_DEFAULT_FDT_FILE="r8a77970-eagle.dtb"
 CONFIG_VERSION_VARIABLE=y
 CONFIG_HUSH_PARSER=y
 CONFIG_CMD_BOOTZ=y
+CONFIG_CMD_ASKENV=y
 CONFIG_CMD_GPIO=y
 CONFIG_CMD_I2C=y
 CONFIG_CMD_SPI=y
diff --git a/configs/r8a77980_condor_defconfig b/configs/r8a77980_condor_defconfig
index eb14f50..1a47ec2 100644
--- a/configs/r8a77980_condor_defconfig
+++ b/configs/r8a77980_condor_defconfig
@@ -18,6 +18,7 @@ CONFIG_DEFAULT_FDT_FILE="r8a77980-condor.dtb"
 CONFIG_VERSION_VARIABLE=y
 CONFIG_HUSH_PARSER=y
 CONFIG_CMD_BOOTZ=y
+CONFIG_CMD_ASKENV=y
 CONFIG_CMD_DFU=y
 CONFIG_CMD_GPIO=y
 CONFIG_CMD_I2C=y
diff --git a/configs/r8a77990_ebisu-4d_defconfig b/configs/r8a77990_ebisu-4d_defconfig
index 9b52a3a..88a9420 100644
--- a/configs/r8a77990_ebisu-4d_defconfig
+++ b/configs/r8a77990_ebisu-4d_defconfig
@@ -16,6 +16,7 @@ CONFIG_DEFAULT_FDT_FILE="r8a77990-ebisu-4d.dtb"
 CONFIG_VERSION_VARIABLE=y
 CONFIG_HUSH_PARSER=y
 CONFIG_CMD_BOOTZ=y
+CONFIG_CMD_ASKENV=y
 CONFIG_CMD_GPIO=y
 CONFIG_CMD_I2C=y
 CONFIG_CMD_MMC=y
diff --git a/configs/r8a77990_ebisu_defconfig b/configs/r8a77990_ebisu_defconfig
index 4b8efdd..45d9bfc 100644
--- a/configs/r8a77990_ebisu_defconfig
+++ b/configs/r8a77990_ebisu_defconfig
@@ -16,6 +16,7 @@ CONFIG_DEFAULT_FDT_FILE="r8a77990-ebisu.dtb"
 CONFIG_VERSION_VARIABLE=y
 CONFIG_HUSH_PARSER=y
 CONFIG_CMD_BOOTZ=y
+CONFIG_CMD_ASKENV=y
 CONFIG_CMD_GPIO=y
 CONFIG_CMD_I2C=y
 CONFIG_CMD_MMC=y
diff --git a/configs/r8a77995_draak_defconfig b/configs/r8a77995_draak_defconfig
index cd205cd..bdf3851 100644
--- a/configs/r8a77995_draak_defconfig
+++ b/configs/r8a77995_draak_defconfig
@@ -17,6 +17,7 @@ CONFIG_DEFAULT_FDT_FILE="r8a77995-draak.dtb"
 CONFIG_VERSION_VARIABLE=y
 CONFIG_HUSH_PARSER=y
 CONFIG_CMD_BOOTZ=y
+CONFIG_CMD_ASKENV=y
 CONFIG_CMD_GPIO=y
 CONFIG_CMD_I2C=y
 CONFIG_CMD_MMC=y
-- 
2.7.4

